components/server/src/ome/services/util/MailUtil.java

```diff
@@ -71,8 +75,7 @@ public String getSender() {
      */
     public void sendEmail(final String from, final String to,
             final String topic, final String body, final boolean html,
-            final List<String> ccrecipients,
-            final List<String> bccrecipients) {
+            final List<String> ccrecipients, final List<String> bccrecipients) {
```

Comment:
I don't suspect anyone is using this, but on the Java side since we don't have optional arguments, it'd be best to duplicate the method, having the old `sendEmail` call the new with an extra null argument.